### ISSUE TITLE
关于训练时间的问题

### ISSUE THREAD

**Issue by Q-Sophia** [bywlzts/CWNet-Causal-Wavelet-Network#14]
作者您好！感谢您愿意开源您的工作！想问一下您在训练比如LOL_v1和LSWR数据集时，训练完全部一次大概是多长时间呢？以及也想问一下那您是使用什么型号的GPU呢？感谢！

**Comment by bywlzts** [comment]
你好，这俩数据集我在单卡4090上差不多一天时间24小时

**Comment by Q-Sophia** [comment]
我也是在4090上训练的，但是我已经训练了30个小时了，LSWR数据集，但是才到92000个迭代，想问一下是我忽略掉什么吗？

**Comment by Q-Sophia** [comment]
我的训练LSWR.yml是这样的
#### general settings全局参数
name: train
use_tb_logger: false
model: enhancement_model
distortion: sr
scale: 1
gpu_ids: [0]
seg: true
clip_type: CLIP

#### datasets训练集路径
datasets:
  train:
    name: ll_dataset
    interval_list: [1]
    random_reverse: false
    border_mode: false

    invention_num: 2
    # your light map path
    lightmap_GT : '/root/autodl-tmp/LSRW-Huawei-20251016T051220Z-1-001/LSRW-Huawei/LightMap/lightmap_huawei/lightmap_huawei'
    # your image path
    dataroot_GT: '/root/autodl-fs/Training_data/Huawei/high'
    dataroot_LQ: '/root/autodl-fs/Training_data/Huawei/low'

    n_workers: 3  # per GPU
    batch_size: 8
    GT_size: 256   # size you want to crop out as input sample.
    LQ_size: 256
    use_flip: true
    use_rot: true
    color: RGB
  val:
    name: ll_dataset
    # your image path
    dataroot_GT: '/root/autodl-fs/Eval/Huawei/high'
    dataroot_LQ: '/root/autodl-fs/Eval/Huawei/low'

    train_size: [600, 400]

#### network structures网络结构配置
network_G:
  which_model_G: CWNet
  nf: 16 # 32 # 64
  n_l_blocks: [1,3,4,3,1]
  n_h_blocks: [1,2,2,2,1]

#### path模型文件路径
path:
  root: ./
  strict_load: false

#### training settings: learning rate scheme, loss优化器参数
train:
  lr_G: 0.0004
  lr_scheme: MultiStepLR
  beta1: 0.9
  beta2: 0.99
  niter: 300000
  lr_steps: [130000, 180000]
  ft_tsa_only: 0
  warmup_iter: -1  # -1: no warm up
  lr_gamma: 0.5
  eta_min: !!float 1e-7
  vgg_loss: false
  pixel_criterion: l1
  pixel_weight: 1.0
  val_freq: !!float 1e3

  manual.resize960x512_seed: 0

#### logger日志与模型保存频率
logger:
  print_freq: 1000
  save_checkpoint_freq: !!float 1000

**Comment by bywlzts** [comment]
niter: 300000  这是总的迭代次数，实际上不需要这么久，你看迭代收敛了就行

**Comment by Q-Sophia** [comment]
噢！好滴谢谢你！然后的话我看LSWR的psnr大概收敛在21.42左右，论文是21.50，这个偏差是属于误差范围是嘛？

**Comment by bywlzts** [comment]
是的 0.08的误差完全在可接受范围内
